docs/source/release/v6.3.0/framework.rst

```diff
@@ -0,0 +1,50 @@
+=================
+Framework Changes
+=================
+
+.. contents:: Table of Contents
+   :local:
+
+.. warning:: **Developers:** Sort changes under appropriate heading
+    putting new features at the top of the section, followed by
+    improvements, followed by bug fixes.
+
+Concepts
+--------
+
+Algorithms
+----------
+
+Data Objects
+------------
+
+Python
+------
+
+
+.. contents:: Table of Contents
+   :local:
+
+.. warning:: **Developers:** Sort changes under appropriate heading
+    putting new features at the top of the section, followed by
+    improvements, followed by bug fixes.
+
```

Comment:
Is this supposed to be here?

Reply:
Yeah, these dev warnings are removed when we tidy up the release notes.